timesketch/frontend/src/components/Sketch/TimeFilter.vue

```diff
@@ -116,6 +117,9 @@ export default {
       this.startDateTime = startDateTimeMoment.format(dateTimeTemplate)
       this.endDateTime = endDateTimeMoment.format(dateTimeTemplate)
 
+      // Change focus to the 'endDateTime' input
+      event.target.parentNode.parentNode.parentNode.getElementsByClassName('input')[1].focus()
```

Comment:
This is a little difficult to follow. If you need to get the DOM element you can use the Ref. See this for an example:
https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements